Documentation/cluster-crd.md

```diff
@@ -80,11 +81,31 @@ A Placement configuration is specified (according to the kubernetes [PodSpec](ht
 - `podAntiAffinity`: kubernetes [PodAntiAffinity](https://kubernetes.io/docs/api-reference/v1.6/#podantiaffinity-v1-core)
 - `tolerations`: list of kubernetes [Toleration](https://kubernetes.io/docs/api-reference/v1.6/#toleration-v1-core)
 
+### Cluster-wide Resources Configuration Settings
+
```

Comment:
could you add an intro paragraph that states why you would want to specify the limits?

Reply:
@travisn Like line 86-87 (https://github.com/rook/rook/pull/1234/files/4baeeef209f58389c097068ef60082e820f3169b#diff-bf1f8fa0e10351d63703924c47efd0e5R87)?